mobile/lib/features/trade/application/trade_values_service.dart

```diff
@@ -24,4 +24,21 @@ class TradeValuesService {
     return rust.api.calculateLiquidationPrice(
         price: price, leverage: leverage.leverage, direction: direction.toApi());
   }
+
+  int getFeeReserve() {
+    // TODO: Fetch from backend
+    // This hardcoded value corresponds to the fee-rate of 4 sats per vbyte. We should relate this value to that fee-rate in the backend.
+    return 1666;
+  }
+
+  int getChannelReserve() {
+    // TODO: Fetch from backend
+    // This is the minimum value that has to remain in the channel. It is defined in rust-lightning and we should fetch this value from the corresponding constant in the backend.
+    return 1000;
+  }
+
+  int getMinTradeMargin() {
+    // This value is an arbitrary number; we only allow trades with a minimum of 1000 sats margin.
+    return 1000;
+  }
```

Comment:
Some of these values are actually not dummy values, but they should be derived based on other values in the backend. I just did not pull that through to move faster 😅

I will go on implementing things, but I'll add a ticket to pull it through - want to avoid spending more time on this.

Reply:
I think that's the definition of all dummy values - we've put them there just to move faster for now 😂